pkg/detectors/ldap/ldap.go

```diff
@@ -98,7 +108,19 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 	return results, nil
 }
 
-func verifyLDAP(ctx context.Context, username, password string, ldapURL *url.URL) bool {
+func isErrDeterminate(err error) bool {
+	switch e := err.(type) {
+	case *ldap.Error:
+		switch e.Err.(type) {
+		case *net.OpError:
+			return false
+		}
+	}
+
+	return true
+}
+
+func verifyLDAP(ctx context.Context, username, password string, ldapURL *url.URL) error {
```

Comment:
nit: I think we can ignore the `ctx` since it's not used in this fxn.
`verifyLDAP(_, context.Context,...)`